src/tests/suite/daos_test.c

```diff
@@ -241,6 +242,13 @@ run_specified_tests(const char *tests, int rank, int size,
 			nr_failed += run_daos_rebuild_simple_test(rank, size,
 						sub_tests, sub_tests_size);
 			break;
+		case 'b':
```

Comment:
The short option should be abbr. of long option, but seems lots of the short options in this file didn't follow the rule.